template/.travis.yml

```diff
@@ -0,0 +1,58 @@
+sudo: false
+
+language: php
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+
+env:
+  global:
+    - COMPOSER_ARGS="--no-interaction"
+    - COVERAGE_DEPS="satooshi/php-coveralls"
+
+matrix:
+  include:
+    - php: 7.1
+      env:
+        - DEPS=lowest
+    - php: 7.1
+      env:
+        - DEPS=locked
+        - CS_CHECK=true
+        - TEST_COVERAGE=true
+    - php: 7.1
+      env:
+        - DEPS=latest
+    - php: nightly
```

Comment:
Last change, I promise! However, I discovered today that we can now use the alias `7.2`! They'll be updating it to track the betas, RCs, and then stable releases. I tested it on the zend-psr7bridge package, and it works brilliantly!